File: backend/HRManagementSystem/src/Core/HRManagementSystem.Domain/Entities/TrainingCourse.cs
namespace HRManagementSystem.Domain.Entities;
public class TrainingCourse : BaseEntity
{
    public required string Title { get; set; }
    public string? Description { get; set; }
    public int? DurationHours { get; set; }

    public ICollection<EmployeeTraining>? EmployeeTrainings { get; set; }
}

File: frontend/src/app/models/training/training-course.ts
export interface TrainingCourse {
  id: number;
  title: string;
  description?: string;
  durationHours?: number;
}

export interface CreateTrainingCourseDto {
  title: string;
  description?: string;
  durationHours?: number;
}

export interface UpdateTrainingCourseDto {
  title?: string;
  description?: string;
  durationHours?: number;
}
